shared/files/server/teamcity-backup

```diff
@@ -7,11 +7,24 @@ TEAMCITY_DIR={{TEAMCITY_DIR}}
 TEAMCITY_DATA_DIR={{TEAMCITY_DATA_DIR}}
 TEAMCITY_USER={{TEAMCITY_USER}}
 TEAMCITY_GROUP={{TEAMCITY_GROUP}}
+TEAMCITY_VAGRANT_DIR={{TEAMCITY_VAGRANT_DIR}}
+
+RESTORE_LATEST=TeamCity_Backup.zip
+BACKUP_DIR=$TEAMCITY_DATA_DIR/backup
+RESTORE_DIR=$TEAMCITY_VAGRANT_DIR/restore
 
 # I don't know why JAVA HOME is not coming through appropriately but it's
 # something to think about
 export JAVA_HOME=/usr/lib/jvm/java-8-oracle
-$TEAMCITY_DIR/bin/maintainDB.sh backup -C -D -L -P -U -A $TEAMCITY_DATA_DIR
-chown -R $TEAMCITY_USER:$TEAMCITY_GROUP $TEAMCITY_DATA_DIR/backup
-chmod 755 $TEAMCITY_DATA_DIR/backup
-chmod 444 $TEAMCITY_DATA_DIR/backup/*.zip
+
+$TEAMCITY_DIR/bin/maintainDB.sh backup --all --data-dir $TEAMCITY_DATA_DIR
+
+# Empty the restore path of the out-dated backup file
+rm $RESTORE_DIR/$RESTORE_LATEST
+
+LAST_BACKUP=$(ls $BACKUP_DIR | tail -n1)
+
+cp $BACKUP_DIR$LAST_BACKUP $RESTORE_DIR/$RESTORE_LATEST
+
+# Thin backups to only keep most recent
+ls -t | awk 'NR>1' | xargs rm -f
```

Comment:
Or we could just delete all files in `$TEAMCITY_DATA_DIR/backup` at this point, since we've moved the latest backup into place in the restore path?

Reply:
yeah, that seems ok to me